docs/source/migration-engine.md

```diff
@@ -7,9 +7,7 @@ Apollo Server provides reporting and persisted queries in native javascript by d
 
 ## Stand-alone Apollo Server
 
-With ENGINE_API_KEY set as an environment variable, Apollo Server creates a reporting agent that sends execution traces to the Engine UI. In addition by default, Apollo Server supports [persisted queries](./features/apq.html).
-
-<!-- FIXME add something about CDN headers-->
+Apollo Server 2 is able to completely replace the Engine Proxy. To enable metrics reporting, add `ENGINE_API_KEY` as an environment variable. Apollo Server will then create a reporting agent that sends execution traces to the Engine UI. In addition by default, Apollo Server supports [persisted queries](./features/apq.html) without needing the proxy's cache. Apollo Server also provides cache-control headers for consumption by a [CDN](./features/cdn.html). Integration with a CDN provides a replacement for the full response caching in Engine Proxy.
```

Comment:
I don't think that last sentence really is true. After all that was already a feature of the Proxy.